packages/runtime/src/controls/rich-text/rich-text.ts

```diff
@@ -74,31 +75,31 @@ export class RichTextControl extends PropController<RichTextControlMessage> {
       case RichTextControlMessageType.CHANGE_BUILDER_EDIT_MODE: {
         switch (message.editMode) {
           case BuilderEditMode.BUILD:
-            this.editor?.deselect().blur()
+            // this.editor?.deselect().blur()
             break
         }
         break
       }
       case RichTextControlMessageType.FOCUS: {
-        this.editor?.focus().moveToRangeOfDocument()
+        // this.editor?.focus().moveToRangeOfDocument()
         break
       }
     }
   }
 
-  setSlateEditor(editor: Editor) {
+  setSlateEditor(editor: Editor, value: RichTextDTO) {
     this.editor = editor
 
     this.send({
       type: RichTextControlMessageType.INITIALIZE_EDITOR,
-      value: editor.value.toJSON({ preserveSelection: false }),
+      value: toRichTextDAO(value),
```

Comment:
I don't know where the translation could go for the Text component / for the propcontroller version

Reply:
This might mean that the component must translate DAO to DTO on initialization. While that is not optimal it might just be required. I prefer the Text component has no notion of different data types and operates solely on DTO.